changelog/v1.6.0-beta1/consul-tls-discovery.yaml

```
@@ -0,0 +1,10 @@
+changelog:
+  - type: NEW_FEATURE
+    issueLink: https://github.com/solo-io/gloo/issues/2544
+    description: |
+      Allow automatic discovery of TLS when using consul services. Requires serveral changes to gloo's helm config to use:
+      Set the settings.integrations.consul.useTls flag to true.
+      Assign a tag for gloo to recognize as the TLS tag if the default (settings.integrations.consul.tlsTagName=glooUseTls) is insufficient.
+      Add the consul root CA for gloo to use at settings.integrations.consul.rootCA
+      Once these are setup in gloo, adding the tlsTagName value as a tag to consul services should cause them to automatically
```

as a user, what is "them" here?
```suggestion
      Once these are setup in gloo, adding the tlsTagName value as a tag to consul services should cause upstreams to automatically
```